api/Startup.cs

```diff
@@ -77,14 +79,19 @@ public void ConfigureServices(IServiceCollection services)
             }); // about cors: should allow request just from my applications
         }
 
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
 
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                FileProvider = new PhysicalFileProvider(Configuration["AppSettings:ImageFolder"]),
+                RequestPath = "/api/hotels/{hotelId}/images"
+            });
```

Comment:
I think you were using the following code style, weren't you?
```suggestion
            app.UseStaticFiles(
                new StaticFileOptions
                {
                    FileProvider = new PhysicalFileProvider(Configuration["AppSettings:ImageFolder"]),
                    RequestPath = "/api/hotels/{hotelId}/images"
                }
            );
```